### PR TITLE
doc: Update docs for 2.4.4 release

### PR DESCRIPTION
**Pull request by umapraseeda**
Update docs for 2.4.4 release

### CODE REVIEW

File: doc/nrf/releases/release-notes-2.4.4.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+See `known issues for nRF Connect SDK v2.4.4`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_244_changelog:
+
+Changelog
+*********
+
+Protocols
+=========
+
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.4.4.rst:78]: Changelog also needs to be updated. FYI @mariusz-nordicsemi  and  @shanthanordic .

File: doc/nrf/releases/release-notes-2.4.4.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``4bbd91a9083a588002d4397577863e0c54ba7038``.
+
+For a complete list of |NCS| specific commits and cherry-picked commits since v2.4.0, run the following command:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^v3.3.99-ncs1
```
Comment (gmarull) [inline doc/nrf/releases/release-notes-2.4.4.rst:91]: this tag won't contain latest additions, right?

Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.4.4.rst:93]: I have kept it the same as https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.4.3.html#zephyr

Comment (gmarull) [reply doc/nrf/releases/release-notes-2.4.4.rst:93]: but we've merged stuff to zephyr this time

Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.4.4.rst:93]: Could you let me know what should be updated in this section? commits etc..

Comment (gmarull) [reply doc/nrf/releases/release-notes-2.4.4.rst:93]: If we tag zephyr, `v3.3.99-ncs2`, right?

File: doc/nrf/releases/release-notes-2.4.4.rst (hunk 1 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.4.4.rst:78]: Anything more to add here for Matter or Bluetooth? Workaround is mentioned here, under Zephyr: 
https://github.com/nrfconnect/sdk-nrf/pull/16940/files#diff-a78b7053d05eb449bb0d66ddb320e15675391a4f431b1d5885b24029fc5bf690R87

File: doc/nrf/releases/release-notes-2.4.4.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Protocols
+=========
+
+
+
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``4bbd91a9083a588002d4397577863e0c54ba7038``.
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.4.4.rst:85]: Hope this commit is correct @gmarull.

**Review by shanthanordic — CHANGES_REQUESTED** [review]
Waiting for final draft before review

File: doc/nrf/releases/release-notes-2.4.4.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+.. _ncs_release_notes_244_changelog:
+
+Changelog
+*********
+
+Protocols
+=========
+
+
+
```
Comment (bjorn-tore-taraldsen) [inline doc/nrf/releases/release-notes-2.4.4.rst:80]: Need to include new net core monitor:
Copy the text from NCS 2.5..0 release notes 
(https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.5.0.html#libraries)

Libraries
This section provides detailed lists of changes by [library](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/index.html#libraries).

Added: 
[Network core monitor](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/others/network_core_monitor.html#network-core-monitor) library for monitoring the status of the nRF5340 processor’s network core.

File: doc/nrf/releases/release-notes-2.4.4.rst [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.4.0 <ncs_release_notes_240>`, :ref:`nRF Connect SDK v2.4.1 <ncs_release_notes_241>`, :ref:`nRF Connect SDK v2.4.2 <ncs_release_notes_242>` and :ref:`nRF Connect SDK v2.4.3 <ncs_release_notes_243>`.
+The changes have an effect on Matter and Bluetooth®.
```
Comment (martintv) [inline doc/nrf/releases/release-notes-2.4.4.rst:20]: I propose adding a line here saying something like 
"This release adds workaround for Anomaly 165 and Anomaly 168 that is affecting the nRF5340 SoC."

**Review by martintv — APPROVED** [review]

File: doc/nrf/releases/release-notes-2.4.4.rst (hunk 5 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.4.4.rst:20]: @bjorn-tore-taraldsen , @shanthanordic please confirm.

Comment (bjorn-tore-taraldsen) [reply doc/nrf/releases/release-notes-2.4.4.rst:20]: I'm okay with this

File: doc/nrf/releases/release-notes-2.4.4.rst (hunk 4 above)
Comment (shanthanordic) [reply doc/nrf/releases/release-notes-2.4.4.rst:83]: This should also go to highlight section.

File: doc/nrfxlib/conf.py [hunk 6]
```diff
@@ -27,7 +27,7 @@
 project = "nrfxlib"
 copyright = "2019-2023, Nordic Semiconductor"
```
Comment (shanthanordic) [inline doc/nrfxlib/conf.py:28]: Should copyright date be updated to 2024

File: doc/nrf/releases/release-notes-2.4.4.rst (hunk 2 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.4.4.rst]: updated to v3.3.99-ncs2

File: doc/nrf/releases/release-notes-2.4.4.rst (hunk 4 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.4.4.rst:86]: Added.

File: doc/nrfxlib/conf.py (hunk 6 above)
Comment (umapraseeda) [reply doc/nrfxlib/conf.py]: Updated the copyright year.

File: doc/nrf/releases/release-notes-2.4.4.rst (hunk 3 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.4.4.rst]: Updated SHA to c0aa4d27d3f2767e88c6c8c4634425086156f7f5 as suggested by @ankuns in the chat.

**Review by gmarull — APPROVED** [review]

File: doc/nrf/releases/release-notes-2.4.4.rst (hunk 1 above)
Comment (shanthanordic) [reply doc/nrf/releases/release-notes-2.4.4.rst:80]: No

**Review by shanthanordic — APPROVED** [review]

**Review by richabp — APPROVED** [review]

File: doc/nrf/releases/release-notes-2.4.4.rst [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``c0aa4d27d3f2767e88c6c8c4634425086156f7f5``.
+It also contains some |NCS| specific additions and commits cherry-picked from the upstream Zephyr repository including the following one:
+
+* Added workarounds in sdk-zephyr for the Anomaly 168 and the Anomaly 165 that are affecting the nRF5340 SoC.
+
+For a complete list of |NCS| specific commits and cherry-picked commits since v2.4.0, run the following command:
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.4.4.rst:99]: since v2.4.0, is this correct @gmarull?

**Review by divipillai — APPROVED** [review]

**Review by bjorn-tore-taraldsen — APPROVED** [review]

**Pull request merged**